redpanda/ci/01-one-node-cluster-values.yaml

```diff
@@ -17,3 +17,6 @@ statefulset:
   resources:
     limits:
       memory: 1Gi
```

Comment:
I believe this could be removed due to the associated lines being removed in `_helpers.tpl`

Reply:
Nope. The kind cluster in Github action has less resources than we would like to have. We can run into issues that Redpanda will not start because of the insufficient memory.